src/vs/workbench/contrib/search/browser/media/searchview.css

```diff
@@ -266,24 +287,50 @@
 	margin-right: 12px;
 }
 
-.pane-body:not(.wide) .search-view > .results > .monaco-list .monaco-list-row:hover .filematch .monaco-count-badge,
-.pane-body:not(.wide) .search-view > .results > .monaco-list .monaco-list-row:hover .foldermatch .monaco-count-badge,
-.pane-body:not(.wide) .search-view > .results > .monaco-list .monaco-list-row:hover .linematch .monaco-count-badge,
-.pane-body:not(.wide) .search-view > .results > .monaco-list .monaco-list-row.focused .filematch .monaco-count-badge,
-.pane-body:not(.wide) .search-view > .results > .monaco-list .monaco-list-row.focused .foldermatch .monaco-count-badge,
-.pane-body:not(.wide) .search-view > .results > .monaco-list .monaco-list-row.focused .linematch .monaco-count-badge {
```

Comment:
Should we enable `css.format.spaceAroundSelectorSeparator` in our codebase?
/cc @aeschli

Reply:
I'm fairly indifferent, @aeschli I think you were the last to edit that line? https://github.com/microsoft/vscode/blob/29b117216229e4cfdd3d95fd4cb83ad4c37529fc/extensions/css-language-features/package.json#L332

Reply:
The current default reflects the defaults as chosen by jsbeautify: https://github.com/beautify-web/js-beautify/blob/12bc378bd465dc8771609c4a181d49462dbf7dd1/tools/template/beautify-css.wrapper.js#LL48-L49C88

But personally, I prefer spaces around selector separators. So I'm not vetoing a change :-)

Reply:
Oh wow, I inadvertently edited @andreamah 's comment instead of adding a new comment. I didn't know I could do that. Sorry @andreamah.

Reply:
@mjbvz do you have a strong opinion about this? Otherwise, I'll just keep it the same I guess.

Reply:
My vote is to not change the formatting since I believe our code mostly includes spaces around the selector separators (it also makes the PR more tricky to review)

Reply:
created https://github.com/microsoft/vscode/issues/169664 to track this